docs/releases/1.11.7.txt

```diff
@@ -13,3 +13,6 @@ Bugfixes
   argument is a callable that returns ``None`` (:ticket:`28601`).
 
 * Fixed the Basque ``DATE_FORMAT`` string (:ticket:`28710`).
+
+* Fixed sending status 500 instead of status 400 when an incorrect `Host`
```

Comment:
I'm not sure that it's important to backport the fix. It seems like it would only affect someone who isn't following [the deployment advice](https://docs.djangoproject.com/en/dev/howto/deployment/checklist/#allowed-hosts), 
> You should also configure the Web server that sits in front of Django to validate the host. It should respond with a static error page or ignore requests for incorrect hosts instead of forwarding the request to Django. This way you’ll avoid spurious errors in your Django logs (or emails if you have error reporting configured that way).